### ISSUE TITLE
Cannot find module 'node:os'

### ISSUE THREAD

**Issue by alukach** [typescript-eslint/tslint-to-eslint-config#1404]
### 🐛 Bug Report

-   `tslint-to-eslint-config` version: 2.12.2
-   ESLint version: 8.10.0
-   Node version: 14.17.3

#### Actual Behavior

```
➤ npx tslint-to-eslint-config
npx: installed 64 in 3.749s
Cannot find module 'node:os'
Require stack:
- /Users/alukach/.npm/_npx/251/lib/node_modules/tslint-to-eslint-config/src/cli/runCli.js
- /Users/alukach/.npm/_npx/251/lib/node_modules/tslint-to-eslint-config/src/cli/main.js
- /Users/alukach/.npm/_npx/251/lib/node_modules/tslint-to-eslint-config/bin/tslint-to-eslint-config
```

#### Expected Behavior

Conversion to work.

#### Reproduction
I'm assuming this is due to #1365

**Comment by alukach** [comment]
Running the following unblocked me:

```
npx tslint-to-eslint-config@2.11.0
```

**Comment by JoshuaKGoldberg** [comment]
Ah, TIL that `node:os` isn't supported in Node 12. Thanks for this @alukach! I'll release a patch version shortly that fixes this import.

**Comment by JoshuaKGoldberg** [comment]
The fix should be released in `tslint-to-eslint-config@2.12.3` 👍
